apps/admin-page/src/app/pages/events/events.component.html

```diff
@@ -0,0 +1,42 @@
+<webstore-item-card title="Events" [content]="content" [isLoading]="isLoading" data-cy="events-card">
+    <ng-template #content>
+        <table mat-table [dataSource]="events" class="mat-elevation-z8 table" data-cy="events-table">
+            <!-- Name Column -->
+            <ng-container matColumnDef="name">
+                <th mat-header-cell *matHeaderCellDef>Name</th>
+                <td mat-cell *matCellDef="let event">{{ event.name }}</td>
+            </ng-container>
+
+            <!-- Updated at Column -->
+            <ng-container matColumnDef="createdAt">
+                <th mat-header-cell *matHeaderCellDef>Created At</th>
+                <td mat-cell *matCellDef="let event">{{ event.createdAt | date }}</td>
+            </ng-container>
+
+            <!-- User Id Column -->
+            <ng-container matColumnDef="userId">
+                <th mat-header-cell *matHeaderCellDef>User Id</th>
+                <td mat-cell *matCellDef="let event">{{ event.userId }}</td>
+            </ng-container>
+
+            <!-- Edit Column -->
```

Comment:
Should we remove this code?

Reply:
as in all the comented out code from the discount page code you got it from :)

Reply:
Resolved in [2e90df3](https://github.com/treecreate/webstore/pull/345/commits/2e90df34940dc01be161422b433f54ef582155af)